ext/node/polyfills/internal/crypto/cipher.ts

```diff
@@ -220,8 +224,8 @@ export class Cipheriv extends Transform implements Cipher {
     return this;
   }
 
-  setAutoPadding(_autoPadding?: boolean): this {
-    notImplemented("crypto.Cipheriv.prototype.setAutoPadding");
+  setAutoPadding(autoPadding = true): this {
+    this.#autoPadding = autoPadding;
```

Comment:
```suggestion
  setAutoPadding(autoPadding): this {
    this.#autoPadding = !!autoPadding;
```

Reply:
Why is this needed if the `autoPadding` argument is of type `boolean`? Is that simply to convert any input into a boolean?

Reply:
> Is that simply to convert any input into a boolean?

Yes